### ISSUE TITLE
Missing labels for nested editables

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor5#11837]
## 📝 Provide detailed reproduction steps (if any)

1.  Open https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html
2.  Launch VoiceOver or other screen reader
3.  Focus the caption of the image.

### ✔️ Expected result

The editable should be announced as an image caption.

### ❌ Actual result

There is no label.

### ❓ Possible solution

The caption can be announced as, e.g. "Caption for \<image's alt> image".

## 📃 Other details

*   Browser: all
*   OS: all
*   First affected CKEditor version: …
*   Installed CKEditor plugins: image, table

The same issue exists for other nested editables, e.g. table cells.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
We don't have any specific a11y markup for the figcaption editable. In a11y tree it's not even visible as an editable element (at least in Google Chrome).

To start off we need to set `role` to a `textbox`. Secondly we need to add a label with the value proposed by @Comandeer (in case there's no alt text on image the label should be simply "Caption for the image").
